coda/coda_mdstore/presentation.py

```diff
@@ -79,6 +82,32 @@ def getFileHandle(codaId, codaPath):
     )
 
 
+def file_chunk_generator(url):
+    """
+    Download a file and stream it
+    """
+    r = requests.get(url, stream=True)
+    if r.status_code != 200:
+        return
```

Comment:
Do you suggest to halt downloading and raise an exception even if one (or few) of the files is/are invalid? or do we want to continue with the download with available files and have an exception raised when all the file urls are invalid to avoid empty bag downloade?

Reply:
I have been looking at this and am not sure of the best path forward. At the moment I am leaning toward leaving it as you have it and leaving it up to the user to verify the bag is valid...not sure how to otherwise handle an exception while streaming the chunks of files...